src/StateSmith/Output/UserConfig/IRenderConfigC.cs

```diff
@@ -33,4 +33,9 @@ public interface IRenderConfigC : IRenderConfig
     /// https://github.com/StateSmith/StateSmith/issues/185
     /// </summary>
     string CEnumDeclarer => "";
+
+    /// <summary>
+    /// Will use &lt;stdbool.h&gt; and bool type instead of int for boolean
+    /// </summary>
+    string UseStdBool => "true";
```

Comment:
Can you please also update https://github.com/StateSmith/StateSmith/blob/main/docs/settings.md#renderconfigc ?